src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedCommentsConnection.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.connection.versioned;
+
+import com.echobox.api.linkedin.client.VersionedLinkedInClient;
+import com.echobox.api.linkedin.types.social.actions.CommentAction;
+import com.echobox.api.linkedin.types.social.actions.CommentResponse;
+import com.echobox.api.linkedin.types.urn.URN;
+
+/**
+ * Comment connection class for posting comments
+ *
+ * @author Sergio Abplanalp
+ *
+ */
+public class VersionedCommentsConnection extends VersionedConnection {
```

Comment:
```suggestion
public class VersionedCommentConnection extends VersionedConnection {
```
It seems we are using singular form for connection classes (except AssetsConnection, which will be deprecated soon)